_config.yml

```diff
@@ -1,6 +1,7 @@
 # [Name of visual theme]
 #theme: jekyll-theme-minimal
 remote_theme: pages-themes/minimal@v0.2.0
+markdown: GFM
```

Comment:
Add documentation comments (a new line between sections)

```markdown
# [Conversion]
markdown: GFM
```

It's a bit wird why use directly `GFM` is a bit different of `kramdown` with `input: GFM` (default). Maybe, to investigate about it and report to Jekyll project @jekyll.

From default config docs: https://jekyllrb.com/docs/configuration/default/
```yml
# Markdown Processors
kramdown:
  auto_ids          : true
  entity_output     : as_char
  toc_levels        : [1, 2, 3, 4, 5, 6]
  smart_quotes      : lsquo,rsquo,ldquo,rdquo
  input             : GFM
  hard_wrap         : false
  footnote_nr       : 1
  show_warnings     : false
```